src/tests/screens/SurveyDetail/detail.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+import '__mocks__/matchMedia';
+import { createMemoryHistory } from 'history';
+import { Router } from 'react-router-dom';
+
+import { DetailsContext } from 'contexts/details';
+import Detail from 'screens/SurveyDetail/detail';
+import SurveyDetailResponse from 'tests/fixtures/surveyDetailResponse.json';
+
+describe('When Detail component is mounted', () => {
+  it('shows the survey image', () => {
+    const history = createMemoryHistory();
+    const { getByAltText } = render(
+      <Router history={ history }>
+        <DetailsContext.Provider value={ SurveyDetailResponse.data }>
+          <Detail />
+        </DetailsContext.Provider>
+      </Router>
+    );
+
+    const surveyImage = getByAltText('SURVEY');
```

Comment:
@rafayet-monon Can we get the element by the class name `survey-detail__image` instead?